docs/guides/admin/docs/releasenotes/subtitles-as-first-class-citizens.txt

```diff
@@ -0,0 +1,87 @@
+- With OC14 we change a lot how subtitles work in Opencast, you may need to migrate!
+
+Subtitles as first class citizens
+---------------------------------------
+With Opencast 14 we want to put more emphasis on subtitles. You can find more details on how subtitles should be
+handled going forward in [Subtitles](../modules/subtitles.md).
+
+This comes with a bit of migration. Namely, subtitles should not be stored as "attachments" anymore, but as "media"
+(as they is called in the Admin UI) or "tracks" (as the are called internally). Therefore, all subtitle files currently
+stored as attachments in your events should be moved to tracks. This can easily be accomplished with the "changetype"
```

Comment:
```suggestion
This comes with a bit of migration. Namely, subtitles should not be stored as "attachments" or "catalogs" anymore, but as "media"
(as they is called in the Admin UI) or "tracks" (as the are called internally). Therefore, all subtitle files currently
stored as attachments or catalogs in your events should be moved to tracks. This can easily be accomplished with the "changetype"
```